selfdrive/car/toyota/values.py

```diff
@@ -296,8 +298,19 @@ class CAR:
 }
 
 # Don't use theses fingerprints for fingerprinting, they are still needed for ECU detection
-IGNORED_FINGERPRINTS = [CAR.RAV4H_TSS2, CAR.HIGHLANDERH_TSS2, CAR.LEXUS_RXH_TSS2, CAR.PRIUS_TSS2,
-                        CAR.LEXUS_NX, CAR.LEXUS_NX_TSS2, CAR.CAMRY_TSS2, CAR.CAMRYH_TSS2, CAR.LEXUS_ESH, CAR.MIRAI]
+IGNORED_FINGERPRINTS = [
```

Comment:
revert the style change, just add to the list

Reply:
Sorry, will do.